### PR TITLE
EFA-582 Display dialog feedback on invite sent

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt [hunk 1]
```diff
@@ -843,6 +845,17 @@ class RecipientPresenter(
         }
     }
 
+    fun shouldDisplayInvitationFeedback(): Boolean =
+        preferences.storage.getBoolean(SHOULD_SHOW_PLANCK_INVITE_SENT_FEEDBACK, true)
```
Comment (Stereodark) [inline k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt:849]: so this is the feature toggle for this invite sent feedback

Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt:849]: yep, it is saved in disk after user selects the checkbox.

File: k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt [hunk 2]
```diff
@@ -860,6 +873,7 @@ class RecipientPresenter(
         private const val PGP_DIALOG_DISPLAY_THRESHOLD = 2
         private const val ZERO_RECIPIENTS = 0
         private const val ONE_ADDRESS = 1
+        private const val SHOULD_SHOW_PLANCK_INVITE_SENT_FEEDBACK = "shouldShowPlanckInviteSentFeedback"
```
[the remark this answers is not in the document]
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt:876]: no no, this is the key for the key-value storage, where we store the boolean.

[the remark this answers is not in the document]
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt]: Renamed adding "_KEY" at the end so it is more clear.

[the remark this answers is not in the document]
Comment (Stereodark) [reply k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt]: ah I see, silly me, I should have checked it was a get and put from a map. Anyways better now, more clear!

**Pull request merged**